src/gui/MItem_crash.hpp

```diff
@@ -89,13 +89,13 @@ class MI_CRASHES_Y : public WI_INFO_t {
 public:
     MI_CRASHES_Y();
 };
-
+    #if HAS_DRIVER(TMC2130)
```

Comment:
Does other trinamics support CRASH_FILTERING? 
Some comment would be nice here

Reply:
I don't know.